packages/terra-navigation-side-menu/src/MenuItem.module.scss

```diff
@@ -107,7 +107,7 @@
       }
 
       .chevron {
-        color: var(--terra-navigation-side-menu-item-selected-chevron-color, #909496);
+        color: var(--terra-navigation-side-menu-item-selected-chevron-color, #1C1F21);
```

Comment:
Was this color provided by the design team?

Reply:
color contrast change reverted. Thanks
f257333e9538e426d69ce329087fe07892bc97da